src/UI/Table.elm

```diff
@@ -1,64 +1,121 @@
 module UI.Table exposing
-    ( Table, table
-    , HeaderRow, headersEnd, header
-    , Row, withStaticRows, rowEnd, cellFromText, cellFromButton
-    , OptRow
-    , CellWidth, withWidth, withCellsWidth, cellWidthEnd, cellWidthPixels, cellWidthPortion
-    , withResponsiveRows, ResponsiveConfig, MobileCover
-    , withCellsDetails, cellMobileDetailsEnd, cellMobileDetailsHide, cellMobileDetailsShow, cellMobileDetailsShowIf
+    ( Table, table, withItems
+    , Columns, columnsEmpty, columnsPushHeader, columnsPush
```

Comment:
I'm not sure about the `Push` term here (in the module in general).

here I would prefer `column`

```elm
            columnsEmpty
                |> column (headerToColumn "Title" |> columnWidthPixels 320)
                |> column (headerToColumn "Author" |> columnWidthPixels 240)
                |> column (headerToColumn "Year" |> columnWidthPixels 120)
```

Also, `columnsPushHeader` is not being used right?

Reply:
In fact, maybe I would change this API as well.

it could be something like: 

`column: String -> ColumnWidth -> Column`, I'm not sure if we will have any other parameter for the column 🤔